unlock-app/src/components/creator/CreatorLock.js

```diff
@@ -71,10 +76,16 @@ export class CreatorLock extends React.Component {
         <Balance amount={lock.balance} />
         {lockComponentStatusBlock}
         {status === 'deployed' && this.state.showEmbedCode &&
-          <LockCode>
+          <LockPanel>
```

Comment:
I actually think it is ok to show an empty list when the lock has not been deployed?